cloud-ts-url-shortener-cache-http/cache.ts

```diff
@@ -1,4 +1,10 @@
-// Copyright 2016-2017, Pulumi Corporation.  All rights reserved.
+// Copyright 2016-2018, Pulumi Corporation.  All rights reserved.
+
+// Note: @pulumi/cloud is a preview package demonstrating how to create cross-cloud pulumi
```

Comment:
// Note: @pulumi/cloud is a preview package demonstrating how to create cross-cloud Pulumi
// components.  If you are targeting a specific cloud like AWS, Azure, or GCP, we recommend you use
// platform-specific packages like @pulumi/aws, @pulumi/azure or @pulumi/gcp.  A platform-specific package
// give you full access to the breadth of its capabilities and comes with many
// abstractions to make development easier.

How about changing it to that?

Reply:
Sure!